app/Http/Controllers/ElectionController.php

```diff
@@ -156,8 +156,12 @@ public function destroy(Election $election)
         return response()->json(array("delete" => $del, "id" => $id));
     }
 
-    public function batchvote(Request $request, $election_id)
+    public function batchVote(Request $request, $election_id)
     {
+        $this->validate($request, [
```

Comment:
Cool, didn't know this was possible to validate like this.

Reply:
It will automatically throw an error 422 with the errors for you.